app/boards/shields/dumbpad/dumbpad.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment to turn on logging, and set ZMK logging to debug output
+# CONFIG_ZMK_USB_LOGGING=y
+# CONFIG_SENSOR_LOG_LEVEL_DBG=y
```

Comment:
I think you mentioned you were going to remove this line, but I still see it 😄

Reply:
Done.